CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add `syncWithoutDetaching` option for BelongsToMany and MorphToMany relationships https://github.com/nuwave/lighthouse/pull/1031
 - Add `injectArgs` option to `@can` directive to pass along client defined
   arguments to the policy check https://github.com/nuwave/lighthouse/pull/1043
+- Dark Mode support for Lighthouse Documentation when user `prefers-color-scheme` is set to `dark` https://github.com/nuwave/lighthouse/pull/1046
```

Comment:
Thanks for taking the time and care to add that in.

The change is great, but it does not belong in `CHANGELOG.md`, only functional changes to the package itself are mentioned there.

Reply:
Makes sense. Removed.